### ISSUE TITLE
Twitch V5 API Deprecation 

### ISSUE THREAD

**Issue by RaithSphere** [Neos-Metaverse/NeosPublic#3645]
### Describe the bug?

Twitch redeem events are not being processed via the API due to dead endpoints.

### Relevant issues

No

### To Reproduce

Spawn twitch chat, use a redeam

### Expected behavior

Twitch Reward events should Impulse

### Screenshots

![Neos_TRiFSNHoqA](![msedge_pbUr7XSiE0](https://user-images.githubusercontent.com/7329135/161370792-846b3049-e0b4-49fc-be18-a555f1c3d1d3.png)://user-images.githubusercontent.com/7329135/161370592-b7c5b1d8-4528-471d-b3a7-85ff181de52f.png)

### How often does it happen?

Always

### Does the bug persist after restarting Neos?

Yes

### Neos Version Number

2022.1.28.1335

### What Platforms does this occur on?

Windows, Linux

### Did this work before?

Yes

### Additional context

Twitch API v5 (i.e. kraken) was decommissioned on February 28, 2022.
https://discuss.dev.twitch.tv/t/legacy-twitch-api-v5-i-e-kraken-shutdown-reminder-february-28-2022/36589

Latest version of the Twitch API is here - https://github.com/TwitchLib/TwitchLib.Api
This would require updates to the TwitchLib.Client.dll and all associated TwitchLib.Api files
https://github.com/TwitchLib/TwitchLib.Api/commit/1610825fbbb91f2e74bb02548b67a0ef0af13389

### Reporters

Raith, Mr_Bubblez, seif1

**Comment by RaithSphere** [comment]
![Neos_TRiFSNHoqA](https://user-images.githubusercontent.com/7329135/161370910-bb2dfa28-951d-4d24-b014-e206a1627339.png)

**Comment by Ruikio** [comment]
Thanks for making this ticket. I was supposed to awhile ago but I had stuff come up.

Yeah the Twitch chat api in Neos no longer picks up redemptions of channel points. Maybe the API was updated and needs to be updated here in Neos? Or maybe it just broke.

Whatever the problem, it sucks. Hopefully this can get looked into.

**Comment by Ruikio** [comment]
While I'm here with github open. I'd like to poke at this.

The API for channel point redemptions being moved means a lot of interactions that I had built into the game have been basically disabled.

Channel points via Twitch allowed me to easily set cooldowns on the website, set the cost (Which inherently is rewarding players for watching.... which also means Neos is also getting some view time) and just plug in the effect easily to Neos.

Without this API, i'm basically forced to do LogiX to set cooldowns, probably set up a bot to type in chat to let the user know the command is still on cooldown for clarity.

A switch to the updated API and letting it do all the hard work seems like the one better investing in.

**Comment by LexiBasilisk** [comment]
Yup- if you haven't noticed, there's still an ongoing dispute, @Ruikio. It'll get sorted in time. It's useful for the official streams, and this issue was written by a team member, so you can pretty much guarantee it'll be looked at in due time.

**Comment by Ruikio** [comment]
Sorry. I had no idea development was halted for a prolonged time due to a dispute.

Jokes aside. I'm glad normal commands work fine for now. Looking forward to when this is changed over though!

Just wanted to lay out more points down is all.
